### ISSUE TITLE
Add note that responses should be smooth and well-sampled

### ISSUE THREAD

**Issue by cdeil** [open-gamma-ray-astro/gamma-astro-data-formats#68]
We should add a note to the current IACT DL3 specs that responses are expected to be smooth and well-sampled, i.e. the burden of e.g. smoothing responses and choosing response sampling is on the DL3 data producer, not the science tools consuming the files.

This has been discussed in detail the the April f2f meeting and I think isn't controversial, so "just" needs someone to make this clear in the current spec. But the question came up in another issue (see https://github.com/open-gamma-ray-astro/gamma-astro-data-formats/issues/67#issuecomment-251415391) in a discussion with @kosack , so I'm splitting it out into this separate reminder issue.

General schemes to handle parameter ranges where the responses are bad (like mask or weight arrays) haven't been investigated, but might ultimately be needed. If you'd like to discuss this, I'd suggest to split it out yet again into a separate (very long-term, large effort) separate issue.

**Comment by jknodlseder** [comment]
I don't think that this should be part of the data format. The format should be independent of any software implementation, and even of any data producer. "Smooth" and "well-sampled" are anyway floppy terms, the accuracy depends at the end on the precision that you want to reach.

**Comment by cdeil** [comment]
I'm about to release v0.2, so moving this issue to v0.3 milestone.

I still think a statement on this point in the DL3 spec would be very important. Especially in CTA there was a misunderstanding in the past year, where the DL3 IRF producers gave histograms with some noise and the DL3 consumers (Gammapy and ctools) had some problems because they do simple linear interpolation / integration on the IRFs to compute reduced responses, that assume that the IRFs are smooth and well sampled.

cc @genotmaier @kosack
